### DISCUSSION TITLE
clash v1.18.1  二进制文件打包后网关无法使用，1.80可以正常

### DISCUSSION THREAD

**Discussion by gukey** [kkkgo/PaoPaoGateWay#60] [问和答]
没事就升级下最新的配置文件，将clash也更新到最新，但是发现1.8.1可以联网，Yacd 界面节点也能显示，测速正常，ping google.com 也能返回 7.0.0.0网段的DNS解析地址，但就是翻墙和谷歌油管都无法使用，不知道最新二进制更新了那些东西，弄得莫名其妙；

**Comment by gukey** [comment]
这地方弄的，https://github.com/MetaCubeX/mihomo/releases ，是不是可以考虑支持下 sing-box 内核，感觉clash兼容性不太靠谱；

**Reply by kkkgo** [reply]
是mihomo的问题，参考 https://github.com/kkkgo/PaoPaoGateWay/discussions/58#discussioncomment-8390187

**Reply by kkkgo** [reply]
但你定制嗅探的话应该不会影响。

**Reply by gukey** [reply]
用最新的 alpha-0c384b1 版本可以，我把嗅探关闭了，一个兼容性问题，测试了快一个小时了；

**Reply by gukey** [reply]
还以为是那配置有问题，一直在找，没想到是内核出问题了。
